wagtail/admin/tests/test_privacy.py

```diff
@@ -364,11 +364,16 @@ def test_explorer_private(self):
         # Check the response
         self.assertEqual(response.status_code, 200)
 
-        # Check the privacy indicator is private
-        self.assertContains(response, '<div class="" data-privacy-sidebar-private>')
-        self.assertContains(
-            response, '<div class="w-hidden" data-privacy-sidebar-public>'
-        )
+        soup = WagtailTestUtils.get_soup(response)
```

Comment:
It's a static method but we can call it via `self` for simplicity:

```suggestion
        soup = self.get_soup(response)
```